apps/privacy-panel/manifest.webapp

```diff
@@ -40,7 +40,8 @@
   },
   "default_locale": "en-US",
   "icons": {
-    "60": "/style/icons/privacy-panel.png"
+    "60": "/style/icons/privacy_panel_60.png",
```

Comment:
I don't see these files in the pull request. Did you mean to add them? I think this would break the icon display.